.github/actions/aurora-manage-cluster/action.yml

```diff
@@ -0,0 +1,202 @@
+        s3-bucket-region: '${{ inputs.s3-bucket-region }}'
+
+        name: '${{ inputs.cluster-name }}'
+        module-name: 'aurora'
+
+        tf-cli-config-credentials-hostname: '${{ inputs.tf-cli-config-credentials-hostname }}'
+        tf-cli-config-credentials-token: '${{ inputs.tf-cli-config-credentials-token }}'
+        tf-terraform-wrapper: '${{ inputs.tf-terraform-wrapper }}'
+
+    - name: Checkout Repository Aurora modules
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
+      with:
+        repository: "camunda/camunda-tf-eks-module"
+        ref: ${{ inputs.tf-modules-revision }}
+        path: ${{ inputs.tf-modules-path }}
+        fetch-depth: 0
+
+    - name: Terraform Init
+      shell: bash
+      id: init
+      working-directory: "${{ inputs.tf-modules-path }}/modules/aurora/"
+      run: |
+        cp ../fixtures/backend.tf ./
+        terraform version
+        terraform init -backend-config="bucket=${{ steps.utility.outputs.TFSTATE_BUCKET }}" -backend-config="key=${{ steps.utility.outputs.TFSTATE_KEY }}" -backend-config="region=${{ steps.utility.outputs.TFSTATE_REGION }}"
+        terraform validate -no-color
+
+    - name: Terraform Plan
+      shell: bash
+      id: plan
+      working-directory: "${{ inputs.tf-modules-path }}/modules/aurora/"
+      run: |
+        terraform plan -no-color -out aurora.plan -var "cluster_name=${{ inputs.cluster-name }}" \
+          -var "engine_version=${{ inputs.engine-version }}" \
+          -var "instance_class=${{ inputs.instance-class }}" \
+          -var "num_instances=${{ inputs.num-instances }}" \
+          -var "username=${{ inputs.username }}" \
+          -var "password=${{ inputs.password }}" \
+          -var "auto_minor_version_upgrade=${{ inputs.auto-minor-version-upgrade }}" \
+          -var 'availability_zones=${{ inputs.availability-zones }}' \
+          -var 'iam_roles=${{ inputs.iam-roles }}' \
+          -var "iam_auth_enabled=${{ inputs.iam-auth-enabled }}" \
+          -var "ca_cert_identifier=${{ inputs.ca-cert-identifier }}" \
+          -var "default_database_name=${{ inputs.default-database-name }}" \
+          -var "vpc_id=${{ inputs.vpc-id }}" \
+          -var 'subnet_ids=${{ inputs.subnet-ids }}' \
+          -var 'cidr_blocks=${{ inputs.cidr-blocks }}'
+
+    - name: Terraform Apply
+      shell: bash
+      id: apply
+      working-directory: "${{ inputs.tf-modules-path }}/modules/aurora/"
+      run: |
+        terraform apply -no-color aurora.plan
+        export aurora_endpoint="$(terraform output -raw aurora_endpoint)"
+        echo "aurora_endpoint=$aurora_endpoint" >> "$GITHUB_OUTPUT"
+
+    - name: Fetch Terraform Outputs
+      shell: bash
+      id: fetch_outputs
+      working-directory: "${{ inputs.tf-modules-path }}/modules/aurora/"
+      run: |
+        all_outputs=$(terraform output -json | jq -c .)
+        echo "all_terraform_outputs=$all_outputs" | tee -a "$GITHUB_OUTPUT"
```

Comment:
no actual change, just a side note.

The cool thing about `tee -a` is that you can append it to multiple files at the same time e.g. `GITHUB_OUTPUT` and `GITHUB_ENV` otherwise it's the same as just `>>`